Sage/carbon
# Accessible sortable table column headers

Sortable columns can now receive focus via the keyboard, and also include:

* `aria-sort` - this attribute is added to the currently sorted column, with its value indicating the sort direction e.g. ascending or descending
* `aria-label` - this is added to the sortable link, and identifies that the column can be sorted, the current sort direction (if applicable), and which sort will be applied when the link is activated.

# Related Issues / Pull Requests
#1142 

# Screenshots / Gifs
![aria-label](https://cloud.githubusercontent.com/assets/52329/26200212/fad3d0c2-3bc4-11e7-9b21-9e9391fb8005.gif)


# Testing Instructions
Can be tested using VoiceOver, and / or inspecting the `aria-sort` and `aria-label` text in the browser dev tools.

**NB** VoiceOver does not seem to pick up changes in the `aria-label` text after sorting, even though you can view the `aria-label` text in the browser dev tools and see it update correctly. Also, if you sort the column, tab off the column header, then tab back on to the column header, the correct text is read out.
